engine/overworld/map_objects.asm

```diff
@@ -2729,11 +2744,11 @@ DEF PRIORITY_HIGH EQU $30
 
 InitSprites:
 	call .DeterminePriorities
-	ld c, PRIORITY_HIGH
+	ld c, PRIORITY_LOW
```

Comment:
Why would low-priority sprites be initialized *before* high-priority ones? Were the constants' names wrong all this time, and this is just a rename? Does this PR have some code change that corresponds to this and somehow makes it okay? Like, if there's a reason this doesn't break anything, I think it should be commented in the code or at least in the PR.

Reply:
OAMs are loaded in reverse order now, so we need to load priorities in reverse order as well. There will be a comment in the code now.